packages/browserify/examples/01-simple-js/index.html

```diff
@@ -1,7 +1,15 @@
 <!DOCTYPE HTML>
 <html lang="en">
 <head>
-    <script src="https://cdn.jsdelivr.net/npm/@lavamoat/snow/snow.js"></script>
+    <script>
+      Object.defineProperty(window, 'SCUTTLER', {
+        value: (globalRef, scuttle) => {
+          console.log('scuttler (start)');
```

Comment:
debug code?

Reply:
nope, just wanted to demonstrate the control you get in the scuttler process when you configure it (seems legit to use in an example)